### ISSUE TITLE
[Adaptive Cards - Designer] - Focus order is not logical on the page on after closing the 'Add an action' and 'Bind...' expandable control using Esc key.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9326]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/
OS Version: 25H2 (OS Build 26200.5761)
Browser Version: Version 145.0.3734.1 (Official build) dev (64-bit)

**Repro Steps:**

1. Open the above URL on the Edge Dev browser.
2. Navigate to the 'Designer' control present on the header section of the page using tab key and activate using enter key.
3. Navigate to the 'Add an action' and 'Bind...' expandable control present in the 'Adaptive Card editor' section using tab key and activate using enter key.
4. Now try to close the expandable control using esc key.
5. Verify that, focus order is logical on the page on after closing the 'Add an action' and 'Bind...' expandable control using Esc key or not.

**Actual Result:**
Focus order is not logical on the page on after closing the 'Add an action' and 'Bind...' expandable control using Esc key. Currently on after closing the expandable flyout using esc key the focus gets lost and on pressing next Tab key press focus lands on the 'Skip to Main Content' control due to which users face difficulty on again navigating to the triggered control.

**Observation:**
Currently user have to press 42 times Tab keystrokes to reach the desired control and while in backward navigation user have to press 19 times Shift + Tab keystrokes.

**Reproducibility:**
Same issue replicates that on after activating the menu items present inside the 'Bind...' expandable control on the page. **Refer Attachment:** Reproducibility_FocusOrder_Bind.

**Expected Result:**
Focus order should be logical on the page on after closing the 'Add an action' and 'Bind...' expandable control using Esc key. After closing the flyout using Esc key focus should not get lost and it should directly return back to the same triggered control i.e., expandable control.

**User Experience:**
Keyboard dependent users will be impacted if the focus order is not logical on the page on after closing the 'Add an action' and 'Bind...' expandable control using Esc key. Due to which users will not be able to perform the required action as it initiates the incorrect flow and user again need to perform extra keystrokes navigate to the required control and might get confused in terms of identifying the focus order of the page.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG22/Understanding/focus-order.html

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/user-attachments/assets/b0d5c82f-e349-497c-917a-7ba693b94f42

![Image](https://github.com/user-attachments/assets/30b0bb17-9eb3-488e-9a9e-435c0ac4c929)

**Reproducibility_FocusOrder_Bind:**

https://github.com/user-attachments/assets/e1d4c44c-657b-4224-8d92-42d7b9a7334f

### Card JSON

```JSON
NA
```
